pslab-core.X/instruments/sensors.h

```diff
@@ -56,5 +60,33 @@
      */
     response_t SENSORS_GetCounter(void);
 
-#endif	/* SENSORS_H */
+    /**
+     * @brief Setup and measure timing from an HCSR04 ultrasonic sensor
+     * 
+     * Pin Connection
+     * ==========================================
+     > VCC  : +5V
+     > TRIG : SQ1
+     > ECHO : LA1
+     > GND  : GND
+     * 
+     * Timing measurement diagram
+     * ==========================================
+     > Trigger Pin (SQR1)
+        _|------|__________________________________
+          <10us>
+        ________|-|-|-|-|-|-|-|-|__________________
+                 <--- Burst --->
+     >  Echo Pin (LA1)
+        ________________________|--------...----|__
```

Comment:
If we are OK with unicode in comments:
```suggestion
        _|‾‾‾‾‾|_______________________________________________
          <10us>
        __________|‾|_|‾|_|‾|_|‾|_|‾|_|‾|_|‾|_|‾|______________
                 <--- Burst --->
     >  Echo Pin (LA1)
        ___________________________________________|‾‾‾...‾‾‾|_
```

Reply:
Looks good!